ir/base.def

```diff
@@ -232,6 +232,8 @@ class Annotations {
 /// Implemented by all objects that can have annotations
 interface IAnnotated {
     virtual Annotations getAnnotations() const = 0;
+    Annotation getAnnotation(cstring name) const override {
```

Comment:
I am wondering whether this should be called getSingleAnnotation - to make it clear that it expects a single annotation with this name.